### ISSUE TITLE
Derivation of gain feature importance

### ISSUE THREAD

**Issue by zzzbobml** [dmlc/xgboost#8755]
Could anyone provide a simple explanation of how the gain feature importance is computed? Particularly it's dependence with learning rate as I'm not finding it easily in the code.

**Comment by trivialfis** [comment]
If the importance type is weight, the importance is the sum of the split count on each feature. If the type is cover, then it's the mean of hessian value, if the type is the gain, it's the mean of loss change for each split.
